src/controllers/user/getallusers.ts

```diff
@@ -0,0 +1,14 @@
+import { NextFunction, Request, Response } from 'express';
+import User from '../../database/user';
+import CustomError from '../../helpers';
+
+const getAllUsers = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+  try {
+    const allUsers = await User.find({}, { password: 0 });
```

Comment:
with select filter, when you want the data without specific property  we add 1 to the property